installers-conda/resources/post-install.bat

```diff
@@ -1,17 +1,23 @@
 :: This script launches Spyder after install
 @echo off
 
-echo %PREFIX% | findstr /b "%USERPROFILE%" > nul && (
-    set shortcut_root=%APPDATA%
-) || (
-    set shortcut_root=%ALLUSERSPROFILE%
+set spy_rt=%PREFIX%\envs\spyder-runtime
+set menu=%spy_rt%\Menu\spyder-menu.json
+set mode=system
+if exist "%PREFIX%\.nonadmin" set mode=user
+
+:: Get shortcut path
+for /F "tokens=*" %%i in (
+    '%PREFIX%\python -c "from menuinst.api import _load; menu, menu_items = _load(r'%menu%', target_prefix=r'%spy_rt%', base_prefix=r'%PREFIX%', _mode='%mode%'); print(menu_items[0]._paths()[0])"'
+) do (
```

Comment:
Is it possible to call a Python script here? If it is, then you could also implement my idea of replacing this code by a similar script than the one I suggested to use for the workflow (it could even be the same?)

The thing is this could grow in complexity in the future, so trying to pack everything in the post install script could make this code hard to maintain.

Reply:
Yes, it is possible. Let me give is some thought regarding the best way to handle it.
For the CI workflow, the script could live just about anywhere in the Spyder repo. For the installer, maybe we could inject it into the `bin` of the base environment, essentially providing our own cli for `menuinst`. Maybe there is a way to use the same script in all scenarios...